### PR TITLE
Fixes line length and javadocs

### PR DESCRIPTION
**Pull request by JLLeitschuh**

### CODE REVIEW

**Comment by SamCarlberg** [comment]
Why change indentation from 4 space to 2?

Why did you remove the import ordering?

**Comment by JLLeitschuh** [comment]
Ordering, is there a way to get intellij to do that automatically?

**Comment by JLLeitschuh** [comment]
> Why change indentation from 4 space to 2?

It makes more sense with the 100 character limit.
Also, its more closer to the google styleguide.

**Comment by SamCarlberg** [comment]
`Intellij > Settings > Editor > Java > Imports`. You'll need to set it manually, the checkstyle plugin doesn't set it from the configuration file.

I prefer 4 spaces to 2, I find it much easier to read. But it does hit that 100-character limit pretty quick

File: checkstyle.xml [hunk 1]
```diff
@@ -16,196 +16,215 @@
...
+            <property name="ignoreFinal" value="true"/>
+            <property name="allowedAbbreviationLength" value="4"/> <!-- Allow 'GRIP' in all caps -->
         </module>
-        <module name="OverloadMethodsDeclarationOrder" />
+        <module name="OverloadMethodsDeclarationOrder"/>
         <module name="VariableDeclarationUsageDistance">
-            <property name="allowedDistance" value="5" />
-        </module>
-        <module name="AvoidStarImport" />
-        <module name="CustomImportOrder">
```
Comment (SamCarlberg) [inline checkstyle.xml:176]: Why did you remove the import ordering? I don't want IDEs to automatically reorder imports and blow up diffs

Comment (JLLeitschuh) [inline checkstyle.xml:176]: Does eclipse order differently?

Comment (SamCarlberg) [inline checkstyle.xml:176]: It's possible for someone to set up their IDE (Eclipse, Intellij, Netbeans, whatever) to have an import order that's not the same as someone else's IDE. Setting an explicit format in checkstyle ensures that nobody will accidentally reformat imports and needlessly change lines
